README.md

```diff
@@ -173,3 +176,4 @@ Reports of bugs and feature requests are also welcome.
 # Credits
 
 The development of SciQLop is supported by the [CDPP](http://www.cdpp.eu/).
+We acknowledge support from the federation [Plas@Par](https://www.plasapar.sorbonne-universite.fr)
```

Comment:
Needs a `<br> ` to keep both lines separated.

Reply:
up rebase